### PR TITLE
adds reconn on close

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

File: services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts [hunk 1]
```diff
@@ -41,73 +44,48 @@ export default class LiveSyncerService {
...
   }
 
   @EventListener('close')
-  onClose(code: number) {
+  async onClose(code: number) {
     this.logger.error(
       `Server at ${this.ws.url} disconnected with code ${code}`
     );
     this.isWsConnected = false;
+    const newConnection = await createWebSocket({
```
[the remark this answers is not in the document]
Comment (rattrap) [reply services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts:73]: If that's the case then maybe we should move back to the original package

File: services/explorer-indexer/src/types/rpcWs/RpcWsResponse.enums.ts [hunk 2]
```diff
@@ -16,9 +16,9 @@ export enum Topics {
...
 export enum Subscriptions {
-  NEW_DAG_BLOCKS = '0x1', // @note fired when a DAG block is accepted by the consensus
-  NEW_DAG_BLOCKS_FINALIZED = '0x2', // @note fired when a DAG block is inserted into a PBFT block
-  NEW_PBFT_BLOCKS = '0x3', // @note fired when a PBFT block is accepted by the consensus
-  NEW_HEADS = '0x4', // @note fired when a PBFT ns "mined": all transactions inside it were executed
-  NEW_PENDING_TRANSACTIONS = '0x5', // @note fired when a tx is registered to the mempool
+  // NEW_DAG_BLOCKS = '0x1', // @note fired when a DAG block is accepted by the consensus
+  // NEW_DAG_BLOCKS_FINALIZED = '0x2', // @note fired when a DAG block is inserted into a PBFT block
+  // NEW_PBFT_BLOCKS = '0x3', // @note fired when a PBFT block is accepted by the consensus
+  NEW_HEADS = '0x1', // @note fired when a PBFT ns "mined": all transactions inside it were executed
```
Comment (rattrap) [inline services/explorer-indexer/src/types/rpcWs/RpcWsResponse.enums.ts:22]: I don't think this is right. `0x1` was `NEW_DAG_BLOCKS`

**Review by rattrap — CHANGES_REQUESTED** [review]

File: services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (VargaElod23) [reply services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts:73]: I had an unpushed line... Sry. https://github.com/Taraxa-project/taraxa-platform/pull/299/files#diff-981f4369d6e7fdaab6916b36856afd864d2fc469c03b79bd89d470a01c9c9e16R84

File: services/explorer-indexer/src/types/rpcWs/RpcWsResponse.enums.ts (hunk 2 above)
Comment (rattrap) [reply services/explorer-indexer/src/types/rpcWs/RpcWsResponse.enums.ts:22]: Oh. Ok. Thought it was the Topics thing

**Review by rattrap — APPROVED** [review]

**Pull request merged**